experimental/generation/generator/packages/library/templates/standard/form-CancelConfirmation-assign-boolean.dialog.lg

```diff
@@ -3,30 +3,25 @@
 # filename
 - ${formDialogFilename('CancelConfirmation-assign-boolean.dialog')}
 
-# template
+# generator
 - ```
 {
     "$schema": "${appSchema}",
     "$kind": "Microsoft.OnAssignEntity",
-    "operation": "Set()",
+    "operation": "Assign()",
     "property": "CancelConfirmation",
     "value": "boolean",
     "actions": [
-        {
-            "$kind": "Microsoft.SetProperty",
-            "property": "$CancelConfirmation",
-            "value": "=@boolean"
-        },
         {
             "$kind": "Microsoft.IfCondition",
-            "condition":"$CancelConfirmation == 'true'",
+            "condition":"@boolean == 'true'",
```

Comment:
why change CancelConfirmation to @boolean

Reply:
Because the I could simplify the logic.